packages/contracts/test/contracts/L1/rollup/CanonicalTransactionChain.gas.spec.ts

```diff
@@ -302,9 +302,9 @@ describe('[GAS BENCHMARK] CanonicalTransactionChain', () => {
       const gasUsed = receipt.gasUsed.toNumber()
 
       console.log('Benchmark complete.')
-      console.log('Gas used:', gasUsed)
+      printGasSavings(gasUsed, 237_065)
```

Comment:
Where exactly are the two hardcoded inputs to `printGasSavings` coming from? Are they relative to `0.4.0` or something?

Reply:
Yes, they are just from me running and copying/pasting them in, you can see them [here too](https://gist.github.com/maurelian/0477e247eee7059bb81820dbde2735d2)
TBH they were mostly useful during my dev process, I'm going to add a commit to remove them.